tests/utils.py

```diff
@@ -178,3 +178,15 @@ def wrapper():
         used.append(item)
         return item
     return wrapper
+
+
+def assert_mutual_in(list_one, list_two):
```

Comment:
There is already `assert_items_equal` for this: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertItemsEqual .

I can change this when I merge.